release-notes/8.0/8.0.1/api-diff/Microsoft.NETCore.App/8.0.1_System.Net.Security.md

```diff
@@ -0,0 +1,21 @@
+# System.Net.Security
```

Comment:
Changes themselves look okay, but the diff looks weird (no body for CipherSuitesPolicy)

Reply:
It's not the ideal way, but I guess it makes sense. The tool considers the addition of new attributes as a change on the API signature. In this case, the CipherSuitesPolicy class and the NegotiateClientCertificateAsync method both got new attributes.